src/sql/platform/connection/test/node/connectionStatusManager.test.ts

```diff
@@ -251,6 +251,7 @@ suite('SQL ConnectionStatusManager tests', () => {
 		newConnection.options['databaseDisplayName'] = newConnection.databaseName;
 		connections.addConnection(newConnection, 'test_uri_1');
 		connections.addConnection(newConnection, 'test_uri_2');
+		newConnection = new ConnectionProfile(capabilitiesService, newConnection);
```

Comment:
previously the newConnection is a mock object, not an actual ConnectionProfile object, the getOptionsKey of it is mocked to always return 'connection1' which caused the test failure.